src/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {FC, ChangeEvent} from 'react';
+import styled from 'styled-components';
+
+import {Box} from '../Box';
+
+import {theme} from '../theme';
+
+interface IFakeInput {
+  checked?: boolean;
+}
+
+type RadioButtonProps = {
+  id: string;
+  label: string;
+  checked?: boolean;
+  value: string;
+  type?: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+};
+
+export const RadioButton: FC<RadioButtonProps> = ({
+  id,
+  type = 'radio',
+  label,
+  checked,
+  value,
+  onChange,
+}) => (
+  <Box flex alignItems="center" mb="24px">
+    <RadioInput
+      id={id}
+      type={type}
+      checked={checked}
+      value={value}
+      onChange={onChange}
+    />
+
+    <RadioLabel htmlFor={id}>
+      <FakeInput checked={checked} />
+      <Text>{label}</Text>
```

Comment:
Can we use the `<Text typo='label>` component here to preserve consistency?
(e.g: similar to labels in "TextInput" or "Dropdown")